include/drivers/dfi/driver.h

```diff
@@ -196,11 +196,22 @@ void dfi_set_init_start_ovr_reg_if(bool override, uint8_t val);
  *
  * @details Get Init Start Status via CSR.
  *
- * @param[out]  init_start  Pointer to return current Init Start Status.
+ * @return  returns status of init_start.
+ * @retval  0 if init_start is low
+ * @retval  1 if init_start is high
+ */
+uint8_t dfi_get_init_start_status_reg_if(void);
+
+/**
+ * @brief   DFI Get Init Complete Status Register Interface
  *
- * @return      void
+ * @details Get Init Complete Status via CSR.
+ *
+ * @return  returns status of init_complete.
+ * @retval  0 if init_complete is high
+ * @retval  1 if init_complete is low
```

Comment:
Not sure if we need to, but I always like to say in the init_complete/init_start, a 0 would be de-asserted, a 1 would be asserted and then in parenthesis add high/low to indicate what logical state that assertion corresponds to.  I think that would indicate very well why low would be a 1 here.